.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: php
 dist: trusty
-sudo: false 
+sudo: false
```

Comment:
This trailing space was removed in https://github.com/mongodb/mongo-php-library/pull/481 when I revised the installation process for MongoDB in `.travis.yml`.